### DISCUSSION TITLE
How to include more columns in the report.pg_matrix.tsv  ?

### DISCUSSION THREAD

**Discussion by WeiqiangChen** [vdemichev/DiaNN#311] [Q&A]
DIANN is great in quantifying a lot of proteins from the DIA raw files. However, there are only 5 columns except for the intensities for each sample in the report.pg_matrix.tsv. There is no way to see if some of these proteins are more confidently quantified.

And in the README.MD, there are many columns in the Description of selected columns in the main report part. 

Is there a way to include all possible columns in this protein groups table?

Is it possible to include NumberOfIdentifiedePeptides, NumberOfQuantifiedpeptides columns? I would like to have some columns to be able to filter for example one-hit-wonders. Scores or Q-Values to filter for more confidently quantified protein groups. 

Thanks a lot!
WeiQiang

**Comment by vdemichev** [comment]
Hi WeiQiang,

Please just match the entries in pg_matrix to the main report or just use the main report. The whole idea of pg_matrix is to provide an output which can be used without any further processing.

Best,
Vadim

**Reply by WeiqiangChen** [reply]
Dear Vadim,

thanks for the reply. 

It is difficult for me to generate a protein groups table from this main report. 

I would like to have something like 
MaxQuant protein groups table (number of peptides identified, contaminant, reverse, sequence coverage, total score), 
also some like Fragpipe(unique intensity, total intensity, unique intensity normalized, total intensity normalized).

Hope you could consider these in a future update. It will help promote the usage of DIA-NN. 

Is the intensity in pg_matrix.tsv  MS1+MS2 normalized intensity? Is there an option in DIA-NN to quantify only unique peptides?

Best!
WeiQiang
